pkg/conditions/repository.go

```diff
@@ -153,6 +153,48 @@ func SetRepositoryFoundConditionToTrue(i interface{}) error {
 	}
 }
 
+func SetValidationPassedToTrue(i interface{}) error {
+	switch in := i.(type) {
+	case invoker.BackupInvoker:
+		return in.SetCondition(nil, kmapi.Condition{
+			Type:    apis.ValidationPassed,
+			Status:  core.ConditionTrue,
+			Reason:  apis.ResourceValidationPassed,
+			Message: fmt.Sprintf("Validation is passed for %s/%s", in.GetObjectMeta().Namespace, in.GetObjectMeta().Name),
```

Comment:
We don't need to mention the resource info here. We are writing the event on the resource itself.